docs/jwst/assign_wcs/reference_files.rst

```diff
@@ -29,53 +29,132 @@ reftype                                     description
 **ote**                Transform through the Optical Telescope Element               NIRSPEC
 **specwcs**            Wavelength calibration models                                 MIRI, NIRCAM, NIRISS
 **regions**            Stores location of the regions on the detector                MIRI
+**v2v3**               Transform from MIRI instrument focal plane to V2V3 plane      MIRI
 **wavelengthrange**    Typical wavelength ranges                                     MIRI, NIRSPEC, NIRCAM, NIRISS
 ===================    ==========================================================   ============================
 
+CRDS Selection Criteria
+-----------------------
 
 CAMERA
-------
+::::::
+CAMERA reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
 
-CRDS Selection Criteria
-:::::::::::::::::::::::
 
-:NIRSPEC: EXP_TYPE
+COLLIMATOR
+::::::::::
+For NIRSPEC, COLLIMATOR reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
 
-Reference File Formats
-::::::::::::::::::::::
+DISPERSER
+:::::::::
+For NIRSPEC, DISPERSER reference files are currently selected based on the values of EXP_TYPE and GRATING in the input science data set.
 
-The camera reference file contains an astropy compound model made up of a polynomial models, rotation and translations. The forward direction is from the FPA to the GWA.
 
-:model: Transform through the CAMERA.
+DISTORTION
+::::::::::
 
-COLLIMATOR
-----------
+For MIRI, DISTORTION reference files are currently selected based on the values of EXP_TYPE, DETECTOR, CHANNEL and BAND in the input science data set.
 
-CRDS Selection Criteria
-:::::::::::::::::::::::
+For FGS, DISTORTION reference files are currently selected based on the values of EXP_TYPE and DETECTOR in the input science data set.
 
-:NIRSPEC: EXP_TYPE
+For NIRCAM, DISTORTION reference files are currently selected based on the values of EXP_TYPE, DETECTOR, CHANNEL, and FILTER in the input science data set.
 
-Reference File Formats
-::::::::::::::::::::::
+For NIRISS, DISTORTION reference files are currently selected based only on the value of EXP_TYPE and PUPIL in the input science data set.
 
-This reference file contains an astropy compound model made up of a polynomial models, rotation and translations. The forward direction is from the GWA to the MSA.
 
-:model: Transform through the COLLIMATOR.
+FILTEROFFSET
+::::::::::::
+For MIRI, FILTEROFFSET reference files are currently selected based on the values of EXP_TYPE and DETECTOR in the input science data set.
```

Comment:
I think FILTEROFFSET is unique to MIRI, so again it might be good to note that along with the type name.